docs/_documentation/fundamentals/data-binding.md

```diff
@@ -559,7 +559,7 @@ Using this syntax, an example binding set is:
 
 ### MvvmCross Defined Custom bindings
 
-In addition to the `Expression` based Fluent bindings, `String` and `Extension Method` based Fluent bindings are also available. This is particularly useful for situations where bindings are needed to View events or to binding targets which are not fully exposed as C# properties. For example, even though a `UIButton` does not have a `Title` property in C#, a 'Title' property can still be set via the use of custom bindings:
+In addition to the `Expression` based Fluent bindings, `String` and `Extension Method` based Fluent bindings are also available. `String` and `Extension Method` based bindings are particularly useful for situations where bindings are needed to View events or to binding targets which are not fully exposed as C# properties. For example, even though a `UIButton` does not have a `Title` property in C#, a `Title` property can still be set via the use of custom bindings:
```

Comment:
I get what you are saying I was just trying to be specific so that no body could read it as including expressions. But I can replace it with "They are".